### DISCUSSION TITLE
How to make vitest watch my fixture files?

### DISCUSSION THREAD

**Discussion by hackwaly** [vitest-dev/vitest#6124] [Q&A] [answered]
I haven't find documentation for this. In my test I read some fixture files from filesystem, I want vitest to re-run after these files changed.

**Comment by sheremet-va** [comment, marked as answer]
This should help: https://vitest.dev/config/#forcereruntriggers

When one of the files changes, Vitest will rerun all tests.

**Reply by hackwaly** [reply]
It's global config. Is there a per suite/test version?

**Reply by sheremet-va** [reply]
No, there is no per-test version for this

**Reply by sheremet-va** [reply]
By the way, if you just need to read files inside the test itself, you can use `import.meta.glob` with `raw` flag. This way Vitest will know what files to rerun when the content changes:

```ts
const files = import.meta.glob('./*.txt', { 
  eager: true, 
  query: '?raw', 
  import: 'default'
})
console.log(files) // { './simple.txt': 'some text' }
```

The same works for a single file:

```ts
import file from './simple.txt?raw'
console.log(file) // 'some text'
```

**Reply by hackwaly** [reply]
Great! That's what I want.
